src/Tracing/SamplingContext.php

```diff
@@ -48,4 +55,24 @@ public function setParentSampled(?bool $parentSampled): void
     {
         $this->parentSampled = $parentSampled;
     }
+
+    /**
+     * Anything that will be provided as an second argument in {@link startTransaction()} will be set here.
```

Comment:
I think a better description would be `Sets additional data that will be provided as a second argument to {@link \Sentry\startTransaction()}.`